conf/application.conf

```diff
@@ -2,14 +2,8 @@ play.crypto.secret="@lERC5v795PcFjt>ouPZF5zPb>@C1s8oDqqN6kYA0S?B8b8Bc]QmbWJLcAY4
 
 play.i18n.langs = [ "en", "de" ]
 
-play.modules.enabled += "play.modules.reactivemongo.ReactiveMongoModule"
 play.modules.enabled += "module.Module"
 
-play.http.filters = "utils.Filters"
-play.http.errorHandler = "utils.ErrorHandler"
```

Comment:
We need this, if we do not handle the authentification errors in all controller subpackges and classes in an alternative way. But also if we do so, it would be much nicer to generalize it and handle it in a global manner.

Reply:
The current implementation never use this ErrorHandler. We replace the function in  WebAppResult